internal/config/options_test.go

```diff
@@ -194,10 +194,10 @@ func Test_parseURLs(t *testing.T) {
 		if (err != nil) != test.wantErr {
 			t.Errorf("Failed to parse URLs %v: %s", test, err)
 		}
-		if o.AuthenticateURL != nil && o.AuthenticateURL.String() != test.authenticateURL {
+		if o.AuthenticateURL != (url.URL{}) && o.AuthenticateURL.String() != test.authenticateURL {
```

Comment:
I converted these straight to default values from nil but I think just get rid of nil check?